docs_user/modules/proc_adopting-compute-services-to-the-data-plane.adoc

```diff
@@ -240,6 +237,27 @@ EOF
 +
 The resources in the `ConfigMap` contain cell-specific configurations.
 
+ifeval::["{build_variant}" == "ospdo"]
```

Comment:
Why don't we use the instructions above for creating the libvirt secret?

Reply:
ack